website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -231,3 +250,12 @@ If you have dbt Core installed locally, either:
 
 You can always uninstall the dbt Cloud CLI to return to using dbt Core.
 </details>
+
+<details>
+<summary>Are there any instructional videos to help guide me through installation? </summary>
```

Comment:
I'm not opposed to this being in the FAQs, but given what we know about scroll rates, this might get more views if we embed it directly into the instructions above, rather than/in addition to putting it here.

Reply:
Yea i also tried adding it in the install header but then it took up too much of the page before introducing the install steps. I could link to it but that also means it'll navigate the user to another tab (and away from the docs). Which is why i figured the faq and a link to the faq was a smoother experience (as opposed to navigating them away from the docs)

Reply:
I'll ask if we can get pop out windows for videos